apps/user-portal/src/configs/globals.ts

```diff
@@ -35,8 +37,9 @@ export let GlobalConfig: RuntimeConfigInterface = {
     clientID: (RUNTIME_CONFIG) ? (RUNTIME_CONFIG.clientID || CLIENT_ID_DEFAULT) : CLIENT_ID_DEFAULT,
     copyrightText: (RUNTIME_CONFIG) ? (RUNTIME_CONFIG.copyrightText || COPYRIGHT_TEXT_DEFAULT) : COPYRIGHT_TEXT_DEFAULT,
     loginCallbackUrl: (RUNTIME_CONFIG) ?
-        ((RUNTIME_CONFIG.serverHost + LOGIN_CALLBACK_URL) || (CLIENT_HOST_DEFAULT + LOGIN_CALLBACK_URL)) :
-        (CLIENT_HOST_DEFAULT + LOGIN_CALLBACK_URL),
+        ((RUNTIME_CONFIG.serverHost + RUNTIME_CONFIG.tenantDomainSeparator + RUNTIME_CONFIG.tenantDomain
```

Comment:
If one of the config is missing it will throw an undefined. Resolve the URL at config level and keep the host simple at this phase.